test/controllers/totals_controller_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class TotalsControllerTest < ActionDispatch::IntegrationTest
+  setup do
```

Comment:
totals controller??  I don't see the controller for this test. please clean up your code before a pr.